utils/Camera.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Were you using this file to test front and back / styling? It doesn't seem to be used anywhere